pkgs/tools/package-management/nix/default.nix

```diff
@@ -218,7 +218,9 @@ in rec {
 
   nix = nixStable;
 
-  nixStable = callPackage common (rec {
+  nixStable = nix_2_4;
+
+  nix_2_3 = callPackage common (rec {
```

Comment:
Thanks for keeping this around for users who need the old Nix command.

Would it be okay to backport the alias so that users who depend on it don't have to do weird hacks for their stable config?

Reply:
The problem is using the `nix_2_3` package name in a config that's used for both 21.05 and 21.11. I'd have to create a workaround for 21.05.

I guess an overlay would solved this but better do this upstream.

Reply:
You could just backport the introduction of `nix_2_3` as an alias to `nixStable` to those releases. Or use an overlay, as you said.